webknossos-datastore/app/com/scalableminds/webknossos/datastore/services/UploadService.scala

```diff
@@ -22,14 +23,31 @@ case class ReserveUploadInformation(uploadId: String,
                                     name: String,
                                     organization: String,
                                     totalFileCount: Long,
+                                    layersToLink: List[LinkedLayerIdentifier],
```

Comment:
Could we make this field optional? This might be an easy change to make the API compatible with old clients.

Reply:
Fair enough, since the client incompatibilities are not checked properly right now, see https://github.com/scalableminds/webknossos-libs/issues/515
Long-term I’d rather not make this optional, since the empty list is the same semantically, and with Option I have to use .getOrElse(List.empty) everywhere.